src/resdk/resources/data.py

```diff
@@ -102,8 +102,15 @@ def __init__(self, resolwe, **model_data):
         self.size = None
         #: scheduled
         self.scheduled = None
-        #: process status - Possible values: Uploading(UP), Resolving(RE),
-        #: Waiting(WT), Processing(PR), Done(OK), Error(ER), Dirty (DR)
+        #: process status - Possible values:
+        #: UP (Uploading - for upload processes),
+        #: RE (Resolving - computing input data objects)
+        #: WT (Waiting - waiting for process since the queue is full)
+        #: PP (Preparing - preparing the environment for processing)
+        #: PR (Processing)
+        #: OK (Done)
+        #: ER (Error)
+        #: DR (Dirty - Data is dirty)
```

Comment:
The explanation here doesn't have any added value.

Reply:
Are you refering just to the line "DR (Dirty - Data is dirty)", or to all of them. What do you propose?

Reply:
I was referring specifically to "Data is dirty". I would remove it and keep the rest.